### PR TITLE
Add Arabic localization

### LINKED ISSUE #266: Help to pull in Arabic Localization
I have finished translating OSRM Text Instructions into the Arabic language in Transifex, So I need help to pull in these changes so I can use them with Mapbox Navigation SDKs.

### PR DESCRIPTION
**Pull request by 1ec5**
Added a complete Arabic localization from Transifex, courtesy of @RazanQ.

@RazanQ, please review the test fixtures to ensure that your translations are joined into correct sentences. Feel free to make corrections at Transifex; this PR will be updated with your translations before merging. Thanks for your contributions!

Fixes #266.

/cc @danpaz

### CODE REVIEW

**Comment by RazanQ** [comment]
I reviewed the test fixtures and made some corrections at Transfix.  
Thanks

**Comment by 1ec5** [comment]
Thanks, I updated the PR with your changes.

File: test/fixtures/v5/arrive/no_modifier.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
         "name": "Street Name"
     },
     "instructions": {
+        "ar": "لقد وصلت إلى الوجهة ",
```
Comment (danpaz) [inline test/fixtures/v5/arrive/no_modifier.json]: Is the trailing whitespace here expected?

Comment (1ec5) [reply test/fixtures/v5/arrive/no_modifier.json]: Here’s the format string:

https://github.com/Project-OSRM/osrm-text-instructions/blob/791335b25f7a6d686c522294ca8a06c7da3064fa/languages/translations/ar.json#L62

`{nth}` resolves to the empty string, but it appears at the end of the format string, so it doesn’t get collapsed down:

https://github.com/Project-OSRM/osrm-text-instructions/blob/b692dfb39b521e91ec6d0207065c91072c3b31a4/index.js#L286

The same issue affects Vietnamese. I guess technically OSRMTI could trim the final message if it causes problems for clients.

**Pull request merged**